src/services/codefixes/codeFixProvider.ts

```diff
@@ -0,0 +1,53 @@
+/* @internal */
+namespace ts {
+    export interface CodeFix {
+        name: string;

```

Comment:
Aren't the error codes handled and the descriptions returned by the function descriptive enough for debugging? :smiley: